drivers/storage/portworx/component/telemetry.go

```diff
@@ -29,60 +29,70 @@ import (
 
 // TODO: change all names to be px specific
 const (
-	// RoleNameSecretManager is name of role secret-manager
-	RoleNameSecretManager = "secret-manager"
-	// RoleBindingNameSecretManager is name of role binding registerrolebinding
-	RoleBindingNameSecretManager = "registerrolebinding"
-	// RoleNameSTCReader is name of role stc-reader
-	RoleNameSTCReader = "stc-reader"
-	// RoleBindingNameSTCReader is name of role binding stcrolebinding
-	RoleBindingNameSTCReader = "stcrolebinding"
-	// ConfigMapNameRegistrationConfig is name of config map registration-config
-	ConfigMapNameRegistrationConfig = "registration-config"
-	// ConfigMapNameProxyConfigRegister is name of config map proxy-config-register
-	ConfigMapNameProxyConfigRegister = "proxy-config-register"
-	// ConfigMapNameProxyConfigRest is name of config map proxy-config-rest
-	ConfigMapNameProxyConfigRest = "proxy-config-rest"
-	// ConfigMapNameTLSCertificate is name of config map tls-certificate
-	ConfigMapNameTLSCertificate = "tls-certificate"
-	// ConfigMapNamePxTelemetryConfig is name of config map px-telemetry-config
-	ConfigMapNamePxTelemetryConfig = TelemetryConfigMapName
-	// DeploymentNameRegistrationService is name of deployment registration-servicea
-	DeploymentNameRegistrationService = "registration-service"
-	// DaemonSetNameTelemetryPhonehome is name of deployment phonehome-cluster
+	// ServiceAccountNameTelemetry is name of telemetry service account
+	ServiceAccountNameTelemetry = "px-telemetry"
+	// ClusterRoleNameTelemetry is name of telemetry cluster role
+	ClusterRoleNameTelemetry = "px-telemetry"
+	// ClusterRoleBindingNameTelemetry is name of telemetry cluster role binding
+	ClusterRoleBindingNameTelemetry = "px-telemetry"
+	// RoleNameSecretManager is name of secret manager role
+	RoleNameSecretManager = "px-telemetry-secret-manager"
```

Comment:
RoleNameSecretManager and RoleNameSTCReader are both bound to px-telemetry service account, in that case we don't need two roles, just put them in one role.